### DISCUSSION TITLE
Dynamic positioning

### DISCUSSION THREAD

**Discussion by anish-palakurthi** [xyflow/xyflow#4640] [Layouting]
### Please describe the feature that you want to propose

Hi - I used xyflow nodes for a project and wrote an algorithm to dynamically calculate absolute positioning values of nodes within groups to generate nested, easy to view layouts. View the PR here: https://github.com/BoundaryML/baml/pull/801 to see the end result and the code. I think this will be useful to many users as defining positioning for parent nodes and children nodes in dynamically generated graphs is a tricky problem to solve.

**Comment by bcakmakoglu** [comment]
As far as I understand it ReactFlow/SvelteFlow are not concerned with layouting.
The library itself is not trying to solve specific use-cases and instead tries to provide you the tools to build this sort of thing yourself, otherwise layouting algos like Elk/Dagre etc. would be integrated but they are not.

Also you marked this as a feature request but I fail to see what exactly you are requesting?
You haven't given a detailed explanation of what the feature is and what the justification would be to integrate it into the core.
Why use your layouting calculations vs any of the layouting algos that are already existing and can do this as well? 🤔

Imo this could just be an example on the docs for people to follow if they want to use this sort of functionality, which you are of course free to create and request to be shown in the docs :)
